### ISSUE TITLE
Throwing out from server when game start

### ISSUE THREAD

**Issue by AcidSoap** [boiscljo/SBA#50]
Plugin version SBW 0.2.3.1
SBA-1.5.12.3 
Client is 1.19.4 
server 1.17.1
Also use viaversion , viarewind at bungeecord.
I have already set uped arena , but when found your addon go to the wiki remove all traders from maps by editing file , then set arena at editor mode and delete all spawners. Then add 4 shoppers for test and try start game with /bw cheat startempty game
I checked logs of my server but is nothing... But fortunately mine is minecraft launched with debug console , when go checking whats happening i found this logs before throwout
```
>> [00:45:11] [Render thread/INFO]: [System] [CHAT] Руины Выбрана команда: Бирюзовые [1/3]
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] [BW] Игра начнётся с пустыми командами. Разрушьте их главные блоки для победы!
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
>> [00:45:13] [Render thread/INFO]: [System] [CHAT]                              Bed Wars
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] 
>> [00:45:13] [Render thread/INFO]: [System] [CHAT]     Защищай свою кровать и уничтожай другие.
>> [00:45:13] [Render thread/INFO]: [System] [CHAT]      Прокачивай себя и свою команду собирая
>> [00:45:13] [Render thread/INFO]: [System] [CHAT]    Железо, Золото, Изумруды и Алмазы с генераторов
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] 
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] 
>> [00:45:13] [Render thread/WARN]: Received packet for unknown team [NPC] fe15a63416: team action: REMOVE, player action: null
>> [00:45:13] [Render thread/WARN]: Received packet for unknown team [NPC] 59f3aaf2f5: team action: REMOVE, player action: null
>> [00:45:13] [Render thread/WARN]: Received packet for unknown team [NPC] 07806dffc9: team action: REMOVE, player action: null
>> [00:45:13] [Render thread/WARN]: Received packet for unknown team [NPC] fecab7c386: team action: REMOVE, player action: null
>> [00:45:13] [Render thread/INFO]: [System] [CHAT] The server you were previously on went down, you have been connected to a fallback server
>> [00:45:13] [Render thread/INFO]: Reloading pipeline on dimension change: NamespacedId{namespace='minecraft', name='bw_arena1'} => NamespacedId{namespace='minecraft', name='overworld'}
>> [00:45:13] [Render thread/INFO]: Destroying pipeline NamespacedId{namespace='minecraft', name='bw_arena1'}
>> [00:45:13] [Render thread/INFO]: Creating pipeline for dimension NamespacedId{namespace='minecraft', name='overworld'}
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
>> [00:45:13] [Render thread/INFO]: Reloading pipeline on dimension change: NamespacedId{namespace='minecraft', name='overworld'} => NamespacedId{namespace='minecraft', name='overworld'}
>> [00:45:13] [Render thread/INFO]: Destroying pipeline NamespacedId{namespace='minecraft', name='overworld'}
>> [00:45:13] [Render thread/INFO]: Creating pipeline for dimension NamespacedId{namespace='minecraft', name='overworld'}
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
>> [00:45:13] [Render thread/INFO]: Stopping worker threads
>> [00:45:13] [Render thread/INFO]: Started 6 worker threads
```

**Comment by AcidSoap** [comment]
Update:
Now i'm tried set up new arena from absollute 0. I have same issue.
Steps for reproduce:
1. Create new arena using SBW 0.2.31
2. Set up 2 teams (for example solo each other)
3. Set up gold, iron spawn
4. Set up iron,emerald spawn
5. Add traders using SBA
6. Save arena
7. Try start arena using cheat startemptygame
Server paper 1.17.1
Minecraft 1.19.4
Citizens-2.0.30-b2900
Before throw out from server you should see something like this
```
>> [20:27:58] [Render thread/INFO]: [System] [CHAT] 
>> [20:27:58] [Render thread/WARN]: Received packet for unknown team [NPC] 1b98d81eea: team action: REMOVE, player action: null
>> [20:27:58] [Render thread/WARN]: Received packet for unknown team [NPC] f52f426470: team action: REMOVE, player action: null
>> [20:27:58] [Render thread/WARN]: Received packet for unknown team [NPC] 88c67caf82: team action: REMOVE, player action: null
>> [20:27:58] [Render thread/WARN]: Received packet for unknown team [NPC] 283dcab81a: team action: REMOVE, player action: null
>> [20:27:58] [Render thread/INFO]: [System] [CHAT] The server you were previously on went down, you have been connected to a fallback server
>> [20:27:58] [Render thread/INFO]: Reloading pipeline on dimension change: NamespacedId{namespace='minecraft', name='bw_arena1'} => NamespacedId{namespace='minecraft', name='overworld'}
```

**Comment by boiscljo** [comment]
This doesn't help as this is User log and not server log. But seems related to a ViaVersion issue translating from 1.17.1 to 1.18. If you play anything on either sides it should work, ex: 1.17 server with 1.12 client, or 1.18.2 server 1.20 client. But It would not work with 1.17 to 1.19 since it passes the error.

**Comment by boiscljo** [comment]
Fixed in 1.5.12.4
